validator/testcases/javascript/predefinedentities.py

```diff
@@ -922,6 +1020,14 @@ def interface_obj(iface):
             "perform unsafe operations. All ctypes use must be carefully "
             "reviewed by a qualified reviewer."),
         "editors_only": True,
+        "signing_help": ("Please try to avoid interacting with or bundling "
+                         "native binaries whenever possible. If you are "
+                         "bundling binaries for performance reasons, please "
+                         "consider alternatives such as Emscripten, "
+                         "JavaScript typed arrays, and Worker threads.",
+                         "Any code which makes use of the `ctypes` API "
+                         "must undergo at least one round of manual code "
+                         "review."),
```

Comment:
This seems to be the exact same message as https://github.com/mozilla/amo-validator/pull/305/files#diff-2398abc7b75c0fb4a06ed98de3770c0aR54, could it be reused?